silvestre-cli/src/commands/info.rs

```
@@ -0,0 +1,80 @@
+
+fn convert_to_silvestre(
+    img: &image::DynamicImage,
+) -> Result<SilvestreImage> {
+    use silvestre_core::ColorSpace;
+
+    let rgba = img.to_rgba8();
+    let (width, height) = img.dimensions();
+    let pixels = rgba.to_vec();
+
+    SilvestreImage::new(pixels, width, height, ColorSpace::Rgba)
+        .map_err(|e| crate::CliError::Silvestre(e))
```

Clippy (1.95.0)

[warning] redundant closure

(warning)